var/spack/repos/builtin/packages/magma/package.py

```diff
@@ -23,7 +23,8 @@ class Magma(CMakePackage, CudaPackage, ROCmPackage):
     test_requires_compiler = True
 
     version('master', branch='master')
-    version('2.6.1', sha256='6cd83808c6e8bc7a44028e05112b3ab4e579bcc73202ed14733f66661127e213')
+    version('2.6.2rc1', commit='5959b8783e45f1809812ed96ae762f38ee701972')
```

Comment:
If this causes a ordering problem with the release of 2.6.2, then we can either remove the rc1 version or mark the 2.6.2 as preferred.  Looks fine.

Reply:
I'm happy to remove 2.6.2rc1 once 2.6.2 proper is released.